db/migrations/1_create_posts.sql

```diff
@@ -0,0 +1,10 @@
+DROP TABLE IF EXISTS posts;
+
+CREATE TABLE posts (
+    id serial PRIMARY KEY,
+    title VARCHAR(100),
+    -- date date DEFAULT GETDATE(),
+    pseudonym VARCHAR(50),
+    body VARCHAR(600)
```

Comment:
600 seems a bit wild for an entry, twitter is 280. tweets sometimes are a little bit short so I would have said 400 max possibly.